### ISSUE TITLE
[simprod-scripts] Detector base class module (Trac #1349)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1341]
http://code.icecube.wisc.edu/projects/icecube/browser/IceCube/projects/simprod-scripts/trunk/python/modules/detectors.py#L54

The tray is executed 100 times in ICBase.  Why?

Migrated from https://code.icecube.wisc.edu/ticket/1349
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:45", 
    "description": "http://code.icecube.wisc.edu/projects/icecube/browser/IceCube/projects/simprod-scripts/trunk/python/modules/detectors.py#L54\n\nThe tray is executed 100 times in ICBase.  Why?", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067105393059", 
    "component": "combo simulation", 
    "summary": "[simprod-scripts] Detector base class module", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-09-16T22:12:42", 
    "milestone": "", 
    "owner": "hdembinski", 
    "type": "rumor / allegation"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150922T01:40:00: olivas changed milestone from "IceSim 4.2" to "IceSim 5.0"

**Comment by icecube-trac** [comment]
Trac update at 20150929T21:34:54: 
* david.schultz changed owner from "david.schultz" to "hdembinski"
* david.schultz commented:

> blame points to Hans.  putting testing code in production?


* david.schultz changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20150929T22:43:24: 
* hdembinski changed resolution from "" to "fixed"
* hdembinski changed status from "assigned" to "closed"
* hdembinski commented:

> In changeset 138071:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="138071"
> i don't remember whether i put it there, but probably, argh, closes #1341
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:45: 
* olivas changed milestone from "IceSim 5.0" to ""
* olivas commented:

> Milestone IceSim 5.0 deleted
